pkg/madmin/README.md

```diff
@@ -286,6 +287,34 @@ Fetches network performance of all cluster nodes using given sized payload. Retu
 | `Error`    | _string_         | Errors (if any) encountered while reaching this node               |
 | `ReadPerf` | _time.Duration_  | Network read performance of the server                             |
 
+<a name="ServerCPUHardwareInfo"></a>
+### ServerCPUHardwareInfo() ([]ServerCPUHardwareInfo, error)
+
+Fetches Mem utilization for all cluster nodes.
+
+| Param             | Type                | Description                                                         |
+|-------------------|---------------------|---------------------------------------------------------------------|
+| `hwi.Addr`        |      _string_       | Address of the server the following information  is retrieved from. |
+| `hwi.Error`       |      _string_       | Errors (if any) encountered while reaching this node                |
+| `hwi.HardwareCPU` | _[]CPUHw.HardwareCPU_ | The CPU hardware info.                                              |
```

Comment:
This doesn't look like correct type please fix.

Reply:
corrected